spec/unit/asset_sync_spec.rb

```diff
@@ -65,6 +65,17 @@
     it "should default log_silently to true" do
       expect(AssetSync.config.log_silently).to be_truthy
     end
+    
+    it "log_silently? should reflect the configuration" do
+      AssetSync.config.log_silently = false
+      expect(AssetSync.config.log_silently?).to eq(false)
+    end
+
+    it "log_silently? should always be true if ENV['RAILS_GROUPS'] == 'assets'" do
+      AssetSync.config.log_silently = false
+      expect(ENV).to receive(:[]).with('RAILS_GROUPS').and_return('assets')
```

Comment:
`expect(ENV)` has the same behavior (stubbing) as `allow(ENV)` but also sets up an expectation that the var will be checked in `ENV`. Just adds more coverage, but I can change it if you would prefer.